index.bs

```diff
@@ -0,0 +1,210 @@
+    1. Call <a>compute the layout shift</a> with <var>unstable elements</var>.
+</div>
+
+Compute the Layout Shift {#sec-compute-layout-shift}
+-----------------------------------------------------
+
+<div algorithm="compute the layout shift">
+    When asked to to <dfn export>compute the layout shift</dfn>, with <var>unstable elements</var>, a list of impacted <a>element</a>'s on the page, as input, run the following steps:
+
+    1. Let <var>impacted region</var> be initially set to <code>null</code>.
+    1. Let <var>max shift fraction</var> be initially set to 0.
+    1. For each tuple <var>tuple</var> in <var>unstable elements</var>:
+        1. Take the geometric union of <var>impacted region</var> with <var>tuple</var>’s <a>currentVisualRepresentation</a> and <var>tuple</var>’s <a>previousVisualRepresentation</a> if <a>previousVisualRepresentation</a> is not null.
+        1. Set <var>max shift fraction</var> equal to <var>shiftFraction</var> if <var>tuple</var>’s <var>shiftFraction</var> > <var>max shift fraction</var>.
+    1. Let <var>impacted fraction</var> be the area of the <var>impacted region</var> divided by the area of the current <a>viewport</a>.
+    1. SEt <var>layout shift</var> to the <var>impacted fraction</var> multiplied by the <var>max shift fraction</var>.  
+    1. If <var>layout shift</var> is greater than 0, create a new {{LayoutShift}} object |newEntry| and set its attributes as follows:
+        1. Set |newEntry|'s {{PerformanceEntry/name}} attribute to <code>"layout-shift"</code>.
+        1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to <code>"layoutShift"</code>.
+        1. Set |newEntry|'s {{PerformanceEntry/startTime}} attribute to <a>current high resolution time</a>.
+        1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to 0.
+        1. Set |newEntry|'s {{LayoutShift/value}} attribute to <var>layout shift</var>.
+    1. <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">Queue the PerformanceEntry</a> |newEntry| object.
+</div>
+
+NOTE: This computation ensures that the layout shift takes into account both the fraction of the viewport that has been impacted by layout stability as well as the greatest impact to any given element in the viewport.  This is to recognize that a large element that moves only a small distance may have a small impact on the perceived stability of the page.
+
+Identify an Unstable Element {#sec-identify-unstable-element}
+--------------------------------------------------------------
+
+<div algorithm="identify an unstable element">
+    When asked to <dfn export>identify an unstable element</dfn>, given <var>element</var> as input, run the following steps:
+
+    1. If <var>element</var>’s <a>previousFrameStartingPoint</a> is null, return false.
+    1. If <a>currentFrameStartingPoint</a> is not at <a>previousFrameStartingPoint</a> and it has changed by more than 3 <a>CSS pixels</a>, return true.
```

Comment:
"is not at" is strange wording, as well as "has changed by more than 3 CSS pixels" (since these are points, not pixels). I'm not sure from this if a movement of 2 pixels vertically and 1 pixel horizontally counts as 3 pixels or not, for example.

Reply:
Does that look better?

Reply:
Yes, thanks! Where it says 'units' here, maybe 'pixel units' is more specific.

Reply:
done.